dev/TabView/TabView.cpp

```diff
@@ -226,6 +219,10 @@ void TabView::OnListViewGettingFocus(const winrt::IInspectable& sender, const wi
 void TabView::OnSelectedIndexPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
 {
     UpdateSelectedIndex();
+    SetTabSeparatorVisibilityState(winrt::unbox_value<int>(args.OldValue()));
+    SetTabSeparatorVisibilityState(winrt::unbox_value<int>(args.OldValue()) - 1);
+    SetTabSeparatorVisibilityState(SelectedIndex() - 1);
+    SetTabSeparatorVisibilityState(SelectedIndex());
```

Comment:
We do this on SelectedIndex and SelectedIndex - 1 because the separator is always on the right of the TabViewItem, so the separators surrounding the selected item are those of it himself and that to the left of himself right?

nit: Short comment explaining why we call with these 4 values

Reply:
yes, you're correct :)
I'll add an explaining comment